test/e2e/shoot/create_and_delete_unprivileged.go

```diff
@@ -35,6 +35,8 @@ var _ = Describe("Shoot Tests", Label("Shoot", "default"), func() {
 	f.GardenerFramework.Config.SkipAccessingShoot = false
 
 	f.Shoot = e2e.DefaultShoot("e2e-unpriv")
+	// TODO(shafeeqes): Use v1.25 Shoot once the PodSecurityConfig has been adapted to use pod-security.admission.config.k8s.io/v1
```

Comment:
Why don't you resolve this TODO immediately?

Reply:
It needs `k8s.io` dependencies v0.25.0.

Reply:
Makes sense - please add this information to the comment.